examples/plugins/http-requests/test-http-requests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+TARGET_IP="127.0.0.1"
+TARGET_PORT="8080"
+
+# This ensures that there is always at least one connection visible in the httpd view.
+# This is a workaround until https://github.com/weaveworks/scope/issues/1257 is fixed.
+/bin/nc $TARGET_IP $TARGET_PORT &
+
+while true; do
+    curl http://$TARGET_IP:$TARGET_PORT &> /dev/null
+    sleep 0.042
```

Comment:
I chose a small sleep just to have a good number of requests show,  I can put 200ms (~=5 request per second).
However, the new plan is to move this plugin to the new [weaveworks-plugins](https://github.com/weaveworks-plugins) organization.